flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/legacy/CurrentJobsOverviewHandler.java

```diff
@@ -69,6 +73,13 @@ public CurrentJobsOverviewHandler(
 	}
 
 	@Override
+	public CompletableFuture<MultipleJobsDetails> handleRequest(HandlerRequest<EmptyRequestBody, EmptyMessageParameters> request, DispatcherGateway gateway) {
+		CompletableFuture<MultipleJobsDetails> jobDetailsFuture = gateway.requestJobDetails(timeout);
+
+		return jobDetailsFuture;
```

Comment:
nit: might as well just make this a one-liner:
`return gateway.requestJobDetails(timeout);`

Reply:
Will change it.